package.json

```diff
@@ -117,6 +117,7 @@
   "scripts": {
     "autostart": "npm-run-all --print-label --parallel watch:compile watch:server",
     "autotest": "supervisor --watch src,test --extensions ts --no-restart-on exit --quiet --exec npm -- test",
+    "benchmark": "mocha --prof --compilers ts:espower-typescript/guess \"test-perf/**/*Benchmark.ts\"; node --prof-process isolate-* > profile.txt",
```

Comment:
It would be good to "scope" this under "test", i.e., "test:benchmark".

Reply:
Also, since the benchmarks are under `test-perf`, is it safe to simplify the glob pattern to just `*.ts`?

Reply:
sure, I thought about doing those things but I was just following the template from microgrammars